extensions/mssql/src/mssql.d.ts

```diff
@@ -319,16 +319,95 @@ declare module 'mssql' {
 	//#region --- Sql Projects
 
 	export interface ISqlProjectsService {
+		addDacpacReference(projectUri: string, dacpacPath: string, suppressMissingDependencies: boolean, databaseVariable?: string, serverVariable?: string, databaseLiteral?: string): Promise<azdata.ResultStatus>;
```

Comment:
Doc comments for new things in this file would be helpful.

Reply:
I'm going to flesh out the conversion tool a little more to generate the C# docstrings, and will update this once I have that working.  Dunno if it'll make this PR.

Reply:
They have made this PR!